### PR TITLE
example: enable to run telegraf automatically

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: examples/telegraf/Dockerfile.spdk [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) Arm Limited and Contributors
+# Copyright (c) Intel Corporation
+FROM fedora:33
```
Comment (mestery) [inline examples/telegraf/Dockerfile.spdk:4]: Any reason to use a Fedora version this old? Fedora 33 was EOL'd on November 30, 2021, as you can see on the [wiki](https://en.wikipedia.org/wiki/Fedora_Linux_release_history).

**Review by mestery — CHANGES_REQUESTED** [review]
This looks good, but one comment on the older Fedora version being used. And I will also open an issue on IPDK to update those containers there as well.

File: examples/telegraf/Dockerfile.spdk (hunk 1 above)
Comment (glimchb) [reply examples/telegraf/Dockerfile.spdk:4]: no reason at all, just used the existing dockerfile from https://github.com/spdk/spdk-csi/blob/master/deploy/spdk/Dockerfile

with new release of SPDK `v22.05` I will re-test and update both SPDK version and Fedora version

Comment (mestery) [reply examples/telegraf/Dockerfile.spdk:4]: FYI: I opened an IPDK issue for this as well, as it seems to use Fedora 33 liberally everywhere and should also be updated: https://github.com/ipdk-io/ipdk/issues/134

**Comment by glimchb** [comment]
I opened #12 to fix it together with new SPDK version, newest Fedora 37 gives me error right now to access mirrors... will investigate regardless of telegraf

**Review by mestery — APPROVED** [review]
Thanks for opening #12 today. I'm good merging this one and then using the issue to track updating Fedora versions.

**Pull request merged**